src/store/actions/index.js

```diff
@@ -5,3 +5,4 @@ export const JOIN_HACK = "JOIN_HACK";
 export const UPDATE_HACK = "UPDATE_HACK";
 export const CLEAR_HACK = "CLEAR_HACK";
 export const FETCH_HACK_A_THON = "FETCH_HACK_A_THON";
+export const LOGINING_IN = "LOGINING_IN";
```

Comment:
LOGINING_IN? LOGGING_IN?

Reply:
I had that debate with myself!

Reply:
Okay... It may look bad but there was a reason. I'll change it